tools/sphere.c

```diff
@@ -37,40 +37,39 @@ extra_init() {
 
 // Apply Slooper and Sloopee shadows, respectively
 int do_sloop( string arg ) {
+    if ( !arg ) {
+        return notify_fail( "What do you want to sloop?" );
+    }
     object slooper = THISP;
-    object shad, shadtwo;
-
-    if ( !arg && !FINDP( arg ) ) {
-        ansi_tell( slooper, "You can only sloop a living being.", BOLD_PURPLE );
-        return 1;
+    if ( !IsWizard( slooper ) ) {
+        return notify_fail( "You cannot summon the power needed." );
+    }
+    object sloopee = FINDP ( arg );
+    if ( !sloopee ) {
+        return notify_fail( "You can only sloop a player." );
+    }
+    if ( slooper == sloopee ) {
+        return notify_fail( "Your head begins to spin and you quickly look away "
```

Comment:
This is a message I could see getting the ANSI treatment since it's keeping in theme with the rest of the object.

Side comment: that you understand to check for this, and also why it works, demonstrates command of the object-oriented thing -- thumbs up